packaging/linux/flatpak/dev.lizardbyte.sunshine.yml

```diff
@@ -8,16 +8,16 @@ sdk-extensions:
 command: sunshine
 separate-locales: false
 finish-args:
-  - --device=all  # access all devices
```

Comment:
Leave two spaces for before comment hashes please... it improves readability and I'm pretty sure the linter will complain if there is only one space